app/views/components/pageflow/admin/users_tab.rb

```diff
@@ -40,7 +40,9 @@ def build(theming)
               end
             end
           end
-          add_membership_button_if_needed(current_user, account, 'account')
+          if authorized?(:add_account_to, :users)
+            add_membership_button(current_user, account, 'account')
+          end
```

Comment:
Shouldn't this check `:add_member_to, account`? In the remaining `:add_account_to` case (in the `UserAccountsTab`), we can then pass the `user` instead of the symbol `:users`. Even if the `UserPolicy` does not rely on the concrete user at the moment, I see no reason to not allow this for the future.

Reply:
For now, I'll leave the symbol, since we still have `can :manage, ::User` in the `ability_mixin`, which covers :add_account_to for sysadmins. I agree with `:add_member_to, account`

Reply:
ok